middleware.md

```diff
@@ -226,7 +226,7 @@ Rarely, you may need your middleware to execute in a specific order but not have
      *
      * This forces non-global middleware to always be in the given order.
      *
-     * @var array
+     * @var string[]
```

Comment:
```suggestion
     * @var class-string[]
```

If you want to be explicit.

Reply:
I agree, but I just wanted to match it with [the line in the kernel](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Foundation/Http/Kernel.php#L71).